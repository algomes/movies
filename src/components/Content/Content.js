import './Content.scss';

import {
    Route,
    Switch
} from "react-router-dom";

import Home from '../../pages/Home';
import Movies from '../../pages/Movies';
import NotFound from '../../pages/NotFound';
import React from 'react';

const Content = () => {
    return (
        <div className="route">
            <Switch>
                <Route exact path="/"><Home /></Route>
                <Route path="/movies"><Movies /></Route>
                <Route path="**"><NotFound /></Route>
            </Switch>
        </div>
    );
  }
  
  export default Content;