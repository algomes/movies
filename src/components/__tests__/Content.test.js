import Content from '../Content/Content';
import {
  BrowserRouter as Router,
} from "react-router-dom";
import { shallow } from 'enzyme';

//Testando com snapshot pois testei o Router no App.js
test('snapshot from Content component', () => {
  const wrapper = shallow(
    <Router>
      <Content/>
    </Router>
  );

  expect(wrapper.debug()).toMatchSnapshot();
});