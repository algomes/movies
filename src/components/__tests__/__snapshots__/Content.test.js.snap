// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot from Content component 1`] = `
"<Router history={{...}}>
  <Content />
</Router>"
`;
